frontend/redirects/default.csv

```diff
@@ -0,0 +1 @@
+/testing-redirects,/,on
```

Comment:
just a test file for now - suggest leaving this here for testing etc, and any new redirects go into a file with the name reflecting the name of the environment. For example: `cclw-redirects.csv`